packages/core/admin/ee/server/services/audit-logs.js

```diff
@@ -103,7 +102,7 @@ const createAuditLogsService = (strapi) => {
       const { user, ...rest } = result;
       return {
         ...rest,
-        user: user ? getService('user').sanitizeUser(user) : null,
+        fullname: user ? `${user.firstname} ${user.lastname}` : null,
```

Comment:
Do you think it would be interesting to have a `function getFullname(user) {...}` in the user service ?